bin/react-docgen.js

```diff
@@ -68,6 +68,11 @@ var ignoreDir = argv.ignore;
 var excludePatterns = argv.exclude;
 var resolver;
 var errorMessage;
+var regexRegex = /\/(.*)\/([igymu]{0,5})/;
```

Comment:
The pattern should be anchored: 

    /^\/(.*)\/([igymu]{0,5})$/;

Otherwise it looks fine to me!